test/Microsoft.Health.Fhir.Ingest.Template.UnitTests/IotJsonPathContentTemplateTests.cs

```diff
@@ -36,6 +35,16 @@ public class IotJsonPathContentTemplateTests
             },
         });
 
+        private static readonly IContentTemplate SingleValueMissingTypeNameTemplate = BuildMeasurementExtractor(new IotJsonPathContentTemplate

```

Comment:
Nit: This field doesn't seem to be referenced in Line 73's test or anywhere else.

Reply:
You are correct; I forgot to delete this. I'll remove it in my next PR